app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsRobot.kt

```diff
@@ -42,12 +42,12 @@ class SettingsRobot {
 private fun assertSettingsView() {
     // verify that we are in the correct library view
     assertBasicsHeading()
-    assertAdvancedHeading()
+    assertPrivacyHeading()
```

Comment:
Is this an unrelated change?

Reply:
> Is this an unrelated change?
No, once we turn on CI, we'd like all the tests to be passing (at least initially).  This test needed a fix while we were waiting on this PR.